specification/hybridcloud/resource-manager/Microsoft.HybridCloud/preview/2023-01-01-preview/hybridCloud.json

```diff
@@ -0,0 +1,1003 @@
+            "$ref": "./examples/CloudConnectionDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "patch": {
+        "tags": [
+          "CloudConnections"
+        ],
+        "operationId": "CloudConnections_UpdateTags",
+        "description": "Updates the specified cloud connection tags.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/cloudConnectionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TagsObject"
+            },
+            "description": "Parameters supplied to update cloud connection tags."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting cloud connection resource.",
+            "schema": {
+              "$ref": "#/definitions/CloudConnection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update Cloud Connections tags": {
+            "$ref": "./examples/CloudConnectionUpdateTags.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "TagsObject": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags."
+        }
+      },
+      "description": "Tags object for patch operations."
+    },
+    "CloudConnectorListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudConnector"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ],
+          "description": "A list of cloud connector resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to hold the next page of results"
+        }
+      },
+      "description": "Response for the ListCloudConnectors API service call"
+    },
+    "CloudConnectorPropertiesFormat": {
```

Comment:
Can it just be named CloudConnectorProperties instead of CloudConnectorPropertiesFormat?

Reply:
Thanks!